index.js

```diff
@@ -139,21 +139,53 @@ const timeout = (promise, ms, abortController) => new Promise((resolve, reject)
 
 const normalizeRequestMethod = input => requestMethods.includes(input) ? input.toUpperCase() : input;
 
+const defaultRetryOptions = {
+	retries: 2,
+	methods: retryMethods,
+	statusCodes: retryStatusCodes,
+	afterStatusCodes: retryAfterStatusCodes
+};
+const normalizeRetryOptions = retry => {
+	if (typeof retry === 'number') {
+		return {
+			...defaultRetryOptions,
+			retries: retry
+		};
+	}
+
+	if (retry.methods && !Array.isArray(retry.methods)) {
+		throw new Error('retry.methods must be an array');
```

Comment:
I noticed now that this should not throw, since Ky returns a promise, it should **always** return a promise and never synchronously throw.